### PR TITLE
The initialization window matches the editor base color or splash background color

### PR DESCRIPTION
**Pull request by CodeLazier**
Now, any window created by godot will be initialized to match the background color set by splash bg, and then the editor mode will be adapted to theme base color, which is to prevent the setting color and theme base color contrast over too large causing visual flicker. Yes,You can have it both ways. (only windows)

https://user-images.githubusercontent.com/14953625/137633490-586d1791-b2dc-4ce4-8987-3b680b623662.mp4

### CODE REVIEW

File: servers/display_server.h [hunk 1]
```diff
@@ -49,6 +49,12 @@ class DisplayServer : public Object {
 		return singleton;
 	}
 
+	//additional parameters,if any,that are passed in when initializing the displayserver.provide space for future expansion.
+	struct ExtraData {
```
Comment (reduz) [inline servers/display_server.h:53]: I think for the time being, I would add a Color and not this struct. If we want to clean this up, we should do it later in a subsequent PR but not left the code in an intermediate state.

**Review by reduz — CHANGES_REQUESTED** [review]

**Comment by reduz** [comment]
Would be very nice to get this one rebased or  salvaged for 4.0.

**Comment by Zireael07** [comment]
Why close?

**Comment by akien-mga** [comment]
I believe @CodeLazier mistakenly pushed the upstream `master` branch on their fork instead of their own rebased `master` branch with this PR's changes. And GitHub automatically closes PRs when their base branch has no difference with upstream (and they can't be reopened).

@CodeLazier You'll have to open a new PR with the contents of https://github.com/godotengine/godot/commit/f82ab2898a109b92818c4287242d783a60ba51b2.

**Pull request closed without merging**